### ISSUE TITLE
Feature Request: ssh-key auth/ credentials per profile

### ISSUE THREAD

**Issue by gaalos** [Ylianst/MeshCentral#3366]
Currently the credentials are saved  with device profile and the only way (i found) to protect multiple credentials is to create device like
device-1-user1
device-1-user2
and allow device-1-user1 to user 1 and device-1-user2 to user2

In user profile maybe add new array contain  password or key that can be used to log in

**Comment by Ylianst** [comment]
Correct, right now the SSH key or password is saved with the device, not with the user. If I make any changes to do, I want to make sure it's designed correctly. Are you saying a user could upload one or more SSH keys to their account and MeshCentral would then give you the option to use these to login?
